tests/js/test_column_formatter.js

```diff
@@ -116,3 +116,72 @@ QUnit.test('empty data', function(assert) {
         '<div class="dt_cell"><div class="dropdown"><div class="dropbtn">0</div></div></div>'
     );
 });
+
+
+QUnit.test('resolve_min_max_value', function(assert) {
```

Comment:
I think a one line comment explaining the rationale of each test could be helpful.